src/Microsoft.Health.Fhir.Core.UnitTests/Features/Export/CreateExportRequestHandlerTests.cs

```diff
@@ -40,7 +45,7 @@ public async void GivenAFhirMediator_WhenSavingAnExportJobSucceeds_ThenResponseS
             var exportOutcome = new ExportJobOutcome(new ExportJobRecord(new Uri(RequestUrl)), WeakETag.FromVersionId("eTag"));
             _fhirDataStore.CreateExportJobAsync(Arg.Any<ExportJobRecord>(), Arg.Any<CancellationToken>()).Returns(exportOutcome);
 
-            var outcome = await _mediator.ExportAsync(new Uri(RequestUrl));
+            var outcome = await _mediator.ExportAsync(new Uri(RequestUrl), DestinationType, ConnectionString);
 
             Assert.NotEmpty(outcome.JobId);
```

Comment:
Should the test verify the other necessary properties are also created correctly?

Reply:
Added an assert to check that `ISecreStore.SetSecretAsync()` was called